pages/index.tsx

```diff
@@ -38,31 +37,22 @@ export const getStaticProps: GetStaticProps = async () => {
   await queryClient.prefetchQuery("all synths", async () =>
     (await Promise.all(cmsSynths.map(fetchCompleteSynth))).filter(errorFilter)
   );
+
   await queryClient.prefetchQuery(
-    "total tvl",
-    async () => await client.getLatestTvl()
+    "total tvm",
+    async () => await client.getLatestTvm()
   );
 
-  const lastTvl = (await queryClient.getQueryData(["total tvl"])) as string;
+  await queryClient.prefetchQuery(
+    "defillama tvl",
+    async () => await getDefillamaTvl()
+  );
 
   await queryClient.prefetchQuery(
-    "total tvm",
-    async () => await client.getLatestTvm()
+    "defillama tvl percent change",
```

Comment:
i think we can probably just keep these defi llama calls named the same, ie 'total tvl change', 'total tvl', and that would remove the need for some of the other changes.

Reply:
Agreed, I changed the naming back to 'total tvl change' and 'total tvl’.